### PR TITLE
Add export task, missing dest / source edge in path node part

### PR DESCRIPTION
**Pull request by zyioump**

### CODE REVIEW

File: opv_tasks/task/exportviewertask.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+                "id_virutaltour_malette": self.virtualtour.id_malette,
+                "id_path_details": path_details.id_path_details,
+                "id_path_details_malette": path_details.id_malette,
+                "path_details": {
+                    "id_path_details": path_details.id_path_details,
+                    "id_malette": path_details.id_malette,
+                    "description": path_details.decription,
+                    "name": path_details.name,
+                    "id_campaign": path_details.campaign.id_campaign,
+                    "id_malette": path_details.campaign.id_malette,
```
Comment (Benvii) [inline opv_tasks/task/exportviewertask.py]: It should be path_details.id_malette
```suggestion
                    "id_malette": path_details.id_malette,
```

File: opv_tasks/task/exportviewertask.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                    "original_id_sensors": path_node.original_id_sensors,
+                    "original_id_malette": path_node.original_id_malette,
+                    "original_gps_pos": path_node.original_gps_pos,
+                    "original_minutes": path_node.original_minutes,
+                    "original_degrees": path_node.original_degrees,
+                    "reconstructed_id_sensors": path_node.reconstructed_id_sensors,
+                    "reconstructed_id_malette": path_node.reconstructed_id_malette,
+                    "reconstructed_gps_pos": path_node.reconstructed_gps_pos,
+                    "reconstructed_minutes": path_node.reconstructed_minutes,
+                    "reconstructed_degrees": path_node.reconstructed_degrees
```
Comment (Benvii) [inline opv_tasks/task/exportviewertask.py]: edge_source and edge_dest ID's list are missing

File: opv_tasks/task/exportviewertask.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+                    "dest_id_path_node_malette": path_edge.dest_path_node.id_malette,
+                    "id_path_details": path_edge.path_details.id_path_details,
+                    "id_path_details_malette": path_edge.path_details.id_malette,
+                    "source_target_yaw_dest": path_edge.source_target_yaw_dest,
+                    "source_yaw_dest": path_edge.source_yaw_dest,
+                }
+
+            self.virtualtour_dict["virtualtour_paths"].append(virtualtour_path_dict)
+        
+        with open(self.virtualtour_dir / "{}-{}.json".format(self.virtualtour.id_virtualtour, self.virtualtour.id_malette), "w") as virtualtour_file:
```
Comment (Benvii) [inline opv_tasks/task/exportviewertask.py]: It's id_malette first to be coherent.
```suggestion
        with open(self.virtualtour_dir / "{}-{}.json".format(self.virtualtour.id_malette, self.virtualtour.id_virtualtour), "w") as virtualtour_file:
```

**Review by Benvii — CHANGES_REQUESTED** [review]
Some changes are needed, also we really need edges in path_node.

File: opv_tasks/task/exportviewertask.py (hunk 1 above)
Comment (zyioump) [reply opv_tasks/task/exportviewertask.py]: We should change key to id_campaign_malette cause id_malette is set before

File: opv_tasks/task/exportviewertask.py [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+                    "name": path_details.name,
+                    "id_campaign": path_details.campaign.id_campaign,
+                    "id_malette": path_details.campaign.id_malette,
+                    "path_nodes": []
+                },
+                "path_edges": {}
+            }
+            path_nodes = self.virtualtour_paths = self._client_requestor.make_all(ressources.PathNodeExtended, filters=(
+                Filter("id_path_details")==path_details.id_path_details,
+                Filter("id_path_details_malette")==path_details.id_malette)
```
Comment (Benvii) [inline opv_tasks/task/exportviewertask.py]: Keep only active nodes (it's the final export of the virtual tour we don't want all nodes).

**Review by Benvii — CHANGES_REQUESTED** [review]

File: opv_tasks/task/exportviewertask.py (hunk 4 above)
Comment (zyioump) [reply opv_tasks/task/exportviewertask.py]: There are no active tag on path node I will just skip disabled node. It's not a problem cause node were created using active panorama.

Comment (Benvii) [reply opv_tasks/task/exportviewertask.py]: yes that what I want

**Review by Benvii — APPROVED** [review]

**Pull request merged**